src/components/MessageList/index.js

```diff
@@ -51,14 +51,16 @@ class MessageList extends React.Component {
             config
         } = this.props;
         return (
-            <ul
+            <div
                 className="ChatContainer-content"
                 ref={ref => (this._ref = ref)}
             >
-                <div className="MessagesList">
+                <ul className="MessagesList">
                     {messages.map((message, i) => (
                         <li
-                            className="MessagesList-item"
+                            className={`MessagesList-item${
+                                message.origin === 'local' ? ' is-local' : ''
```

Comment:
What does `is-local` mean?

Reply:
This is just a class on the element. It's a convention being used elsewhere. Basically, it means the source of the message is from the user and not the bot.